src/frontend/src/components/common/Filter/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState } from 'react';
+import {
+  MinistryList,
+  PiaDrafterFilterList,
+  PiaStatuses,
+  PiaStatusList,
+} from '../../../constant/constant';
+import { isMPORole } from '../../../utils/helper.util';
+import Dropdown from '../Dropdown';
+import { IFilter } from './interfaces';
+
+const Filter = ({ id }: IFilter) => {
+  const [ministry, setMinistry] = useState<string>('');
+  const [status, setStatus] = useState<string>('');
+  const [drafterFilter, setDrafterFilter] = useState<string>('');
+
+  const handlePiaStatusChange = (newStatus: any) => {
+    setStatus(newStatus.target.value);
+  };
+
+  const handleMinistryChange = (newMinistry: any) => {
+    setMinistry(newMinistry.target.value);
+  };
+
+  const handleDrafterFilterChange = (newDrafterFilter: any) => {
+    setDrafterFilter(newDrafterFilter.target.value);
+  };
+
+  const handleClearFilterClick = () => {
+    setStatus('');
+    setMinistry('');
+    setDrafterFilter('');
+  };
+
+  return (
+    <div className="d-flex me-auto">
+      <div className="mt-2"> Filter by</div>
+      <Dropdown
```

Comment:
can you please double check if this dropdown for status change is also only needed when the user is an MPO?

Reply:
yes, I check the design, if the user is a drafter, this user only can see filter by status, but if the user is a MPO role, the user should see all three filter options.